src/mono/mono/mini/interp/transform.c

```diff
@@ -4606,6 +4606,14 @@ interp_emit_sfld_access (TransformData *td, MonoClassField *field, MonoClass *fi
 			}
 			interp_ins_set_dreg (td->last_ins, td->sp [-1].var);
 		} else {
+			// if (G_UNLIKELY (m_field_is_from_update (field)))
```

Comment:
This should be uncommented? Or are you just checking if doing it for all fields is good enough?

Reply:
I just wanted to ensure that this emit pattern is thoroughly tested since I'm not sure what kind of hot reload tests we have, if any.